### ISSUE TITLE
Issue with Installer

### ISSUE THREAD

**Issue by TH3DoCtOr11th** [composer/windows-setup#87]
I encountered an issue while trying to install Composer. It stops at "Checking command-line PHP". I was reading the suggested logs, and it seems I have some problems with the ini. If someone could guide me in troubleshooting, that would be much appreciated. Thanks in advance.
[Setup Log 2018-09-12 #003.txt](https://github.com/composer/windows-setup/files/2377027/Setup.Log.2018-09-12.003.txt)
[Setup Log 2018-09-12 #004.txt](https://github.com/composer/windows-setup/files/2377028/Setup.Log.2018-09-12.004.txt)
[Setup Log 2018-09-12 #005.txt](https://github.com/composer/windows-setup/files/2377029/Setup.Log.2018-09-12.005.txt)
[Setup Log 2018-09-12 #006.txt](https://github.com/composer/windows-setup/files/2377030/Setup.Log.2018-09-12.006.txt)
[Setup Log 2018-09-12 #001.txt](https://github.com/composer/windows-setup/files/2377031/Setup.Log.2018-09-12.001.txt)
[Setup Log 2018-09-12 #002.txt](https://github.com/composer/windows-setup/files/2377032/Setup.Log.2018-09-12.002.txt)

**Comment by johnstevenson** [comment]
Yes, it looks like an ini problem, but what is confusing here is that sometimes PHP's warning output about already loaded extensions is reported, and sometimes it is not. Also, PHP returns a non-zero exit code when it runs a simple test script, which it should not do even if there are extension problems.

Have you got ANSICON installed? This can cause be the cause of non-zero exit codes, but we already check for this (but only in your system directory).

To help understand the ini problems, run `C:\xampp\php\php.exe --ini` which will show you the loaded and scanned ini files. If there is only one, then you have duplicate extension settings in that file. If there is more than one, then the duplicate settings are in one of these.

**Comment by johnstevenson** [comment]
@TH3DoCtOr11th Any news on this? I am updating Composer-Setup and would like to resolve this now, in case something needs fixing.

**Comment by TH3DoCtOr11th** [comment]
I am really sorry for my absence. I have been working non stop on University projects, as well as personal ones. In the end, I solved the problem by checking the log files. It appears there were redundant lines in the php.ini. The strange thing is, I added those lines myself to try and solve the problem. When I commented them, instead of deleting them, it started working. Again, I am really sorry I didn't answer.

**Comment by johnstevenson** [comment]
@TH3DoCtOr11th Thanks (and no worries).

So to confirm, you had a single php.ini (which you managed to bugger up yourself!) and no ANSICON?

**Comment by TH3DoCtOr11th** [comment]
Single php.ini, no ANSICON. But let me be clear. When I tried to install Composer the first time, the php.ini was left untouched, and it didn’t work, infinitely checking the ini. Then, googling it up, i found that different users had two or three lines in their php.ini that mine had not. Since they said that could be the reason for Composer to loop, I addestrato those same lines, but the problem was not Silvester. Then, when you pointed out those log diles, I checked them and found out there were some redundant lines, so I commented them, and finally I could install Composer.
